examples/rust/basic/README.md

```diff
@@ -1,6 +1,6 @@
 # Basic example (Rust)
 
-This example contains a [backend canister](./src/basic_backend/) written in Rust that sends HTTP requests via the [Proxy canister](../../../src/proxy_canister/) to external APIs.
+This example contains a [backend canister](./src/basic_backend/) written in Rust that sends HTTP requests via the [Proxy canister](../../../src/proxy_canister/) to external APIs. Current proxy canister id: `iustv-tiaaa-aaaao-a3aga-cai`.
```

Comment:
On line 34 we already mention that the script will read the Proxy canister id from the `canister_ids.json` file in the root of the repo.
So, I'd rather specify the mainnet canister id there, or even change the **Import the Proxy canister types in your project** section to something like **Use the Proxy canister in your project** and include the canister id there, mentioning that it can also be found in the root `canister_ids.json` file.
@q2333gh what do you think?

Reply:
Oh yes. Since Proxy canister id  is mentioned in canister_ids.json. Thats useful. I will close the PR . and read more about the documentation of this repo ~